tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -46,6 +46,7 @@
 
 import rosgraph.names
 import rospkg
+import loader
```

Comment:
>  If you use the from syntax it needs to be absolute

Not sure what you mean by that, but thanks for pointing out to me that PEP8 says _"Standard library code should avoid complex package layouts and always use absolute imports"_